modules/nodes-containers-init-creating.adoc

```diff
@@ -23,14 +23,14 @@ metadata:
 spec:
   containers:
   - name: myapp-container
-    image: registry.access.redhat.com/ubi8/ubi:latest
+    image: registry.access.redhat.com/ubi9/ubi:latest
```

Comment:
@sheriff-rh registry.access.redhat.com resolves to catalog.redhat.com, but that may not be important at this time?

Reply:
That's an interesting point to pursue at another time. I seem to remember the registry changing name recently. Thanks.